plugins/database/mysql/mysql_test.go

```diff
@@ -46,30 +52,45 @@ func TestMySQL_Initialize(t *testing.T) {
 		"max_open_connections": "5",
 	}
 
-	_, err = db.Init(context.Background(), connectionDetails, true)
+	initReq = newdbplugin.InitializeRequest{
+		Config:           connectionDetails,
+		VerifyConnection: true,
+	}
+
+	db = new(false)
+	_, err = db.Initialize(context.Background(), initReq)
```

Comment:
If you want, you can use the sdk/database/newdbplugin/testing/ package for some of these. It has assertion functions for each of the DB functions such as [`AssertInitialize`](https://github.com/hashicorp/vault/blob/master/sdk/database/newdbplugin/testing/test_helpers.go#L25) which takes a `*testing.T` object, a DB instance, and a request and does all of the boilerplate around running it (creating a context and asserting that the function did not error)

Reply:
I'll keep that in mind for the next DB I do and utilize it then. If we come back to it, I'll redo this one. Thanks for the tip though!